cmd/bundle/validate.go

```diff
@@ -2,15 +2,122 @@ package bundle
 
 import (
 	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"text/template"
 
 	"github.com/databricks/cli/bundle"
 	"github.com/databricks/cli/bundle/phases"
 	"github.com/databricks/cli/cmd/bundle/utils"
 	"github.com/databricks/cli/cmd/root"
-	"github.com/databricks/cli/libs/log"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/fatih/color"
 	"github.com/spf13/cobra"
 )
 
+var validateFuncMap = template.FuncMap{
```

Comment:
We have a superset of this mapping in the `cmdio` library. Can you make that a public global and use that instead?